HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -117,126 +130,132 @@ public void onConnected() {
 
 	public void onError(Exception e) {
 		Toast.makeText(
-                MainActivity.this, 
-                e.getMessage(), 
-                Toast.LENGTH_LONG).show();
+				MainActivity.this,
+				e.getMessage(),
+				Toast.LENGTH_LONG).show();
 	}
 
 	@Override
 	public boolean onOptionsItemSelected(MenuItem item) {
-	    switch (item.getItemId()) {
-	        case R.id.action_connect:
```

Comment:
nit: seems like lots of the indentation moved from 4 spaces to 8. was this intended?